.github/workflows/ci-cd-pipeline.yml

```diff
@@ -121,7 +135,11 @@ jobs:
 
       - name: Promote to Production
         run: |
-          jf rt dpr juno/${{ env.DOCKER_IMAGE_TAG }} ${{ env.REPO_STAGING }} ${{ env.REPO_PROD }}
+          jf rt dpr juno/${{ needs.build_docker_image.outputs.DOCKER_IMAGE_TAG }} ${{ env.REPO_STAGING }} ${{ env.REPO_PROD }}
+
+      - name: Verify Deployment Version (Production)
```

Comment:
Do we need this step for prod? It's gonna failed anyway as for prod we deploy manually

Reply:
I don't know.  How can we make sure the `prod-starknet-rs-tests` and the `prod-starknet-js-tests` are running on the correct version?

I think this will fail because the token might not have permission to deploy to prod anyways, but I think the deployment shouldn't be manually in the long run.  Any reason why it's manually right now?

Reply:
Prod deployment might need additional config, like adding or removing flags. So we'll just push the image to the prod Artifactory but won't expect it to be deployed automatically. It will be updated manually by the DevOps team. 
cc: @gehlotanish @Manjeet-Nethermind 

It'd be great if we could trigger prod tests steps manually or after approval, not automatically after the "promote to prod" step

Reply:
okay, let me fix it then